### ISSUE TITLE
Expected WCAG 3.0 timeline needs updating

### ISSUE THREAD

**Issue by brewerj** [w3c/wai-intro-wcag#130]
Looking at the [WCAG 3 Introduction](https://www.w3.org/WAI/standards-guidelines/wcag/wcag3-intro/), it appears that several things on this page could benefit from updates, but minimally the timeline which currently indicates was last updated 7 Dec 2021 [Timeline (updated 7 December 2021)](https://www.w3.org/WAI/standards-guidelines/wcag/wcag3-intro/#timeline-updated-7-december-2021). Can @rachaelbradley or @michael-n-cooper provide input to @shawna-slh to help with this, or can Michael update it directly? Please ping back to me by email as well as GH if needed, thanks.

**Comment by shawna-slh** [comment]
The [last publication was 07 December 2021](https://www.w3.org/TR/wcag-3.0/) so this is accurate, afaik. Therefore, I'm closing this issue.

Feel free to submit pull requests or issues with other updates.

**Comment by brewerj** [comment]
@shawna-slh : Clarifying, my comment wasn't about the last publication date, it was about updated information regarding the WCAG 3.0 timeline. 

There is some information available on that, linked from the [AGWG home page](https://www.w3.org/WAI/GL/), over to their [timelines Wiki](https://www.w3.org/WAI/GL/wiki/Timelines), including the [WCAG 3.0 project plan](https://github.com/w3c/silver/wiki) which includes a draft milestones schedule. 

I had asked @rachaelbradley and @michaelncooper to help with input on what to consider including in updated messaging on the WCAG Intro page, since I expect that the draft timeline details need some filtering. Please re-open so we can discuss and provide transparency on current plans; thanks.

**Comment by shawna-slh** [comment]
@brewerj I followed all those links and more. It looks like good comprehensive information for people want such level of detail. I didn't find anything that would be appropriate for the [WCAG 3 Introduction page](https://www.w3.org/WAI/standards-guidelines/wcag/wcag3-intro/#timeline-updated-7-december-2021). Nor do I see wording on that page about the timeline that we likely want to change at this point.

Perhaps this issue should go elsewhere, rather than this repo? Regardless, I reopened as requested.
